doc/CITATION.cff

```diff
@@ -0,0 +1,35 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: icepyx
+message: >-
+  If you use this software, please cite the software version
+  you used by its version-specific DOI.
+  The DOI for each version is available on Zenodo (link below).
+type: software
+authors:
+  - family-names: The icepyx Developers
+repository-code: 'https://github.com/icesat2py/icepyx'
+url: 'https://icepyx.readthedocs.io/en/latest/'
+repository: 'https://zenodo.org/record/7754482'
```

Comment:
I'd recommend setting the date and using the DOI so that the automatically generated citation will contain the year and DOI.

```suggestion
date-released: 2023-03-21
doi: 10.5281/zenodo.7754482
```

Reply:
This should have been: repository: 'https://zenodo.org/record/7729175'

I intentionally didn't set any dates or include a version-specific DOI (or meant not to, but clearly accidentally did!) because of the circularity issue [discussed here](https://github.com/xarray-contrib/cf-xarray/issues/398). If the bot gets created, I think it would be great to have the dates and "most recent version" citation be in the CITATION.cff file.

Reply:
Ah yes, the circularity issue. There's a way to 'reserve' a DOI before publishing a Zenodo release, but then you would need to manually do the .zip file upload instead of having the bot do it automatically when a tag is created on GitHub. Pointing to the permanent concept DOI works too.

Reply:
"manually" being the operative word there. ;)